/**:
  ros__parameters:
    camera_name: "camera_2"
    frame_id: "camera_top/camera_link"
    pixel_format: bgr8
    serial_no: 220300108
    camera_info_url: "file:///home/nilay/workspace/tmp/arena_cam_ws/src/arena_camera/config/camera1_info.yaml"
    fps: 5
    horizontal_binning: 1
    vertical_binning: 1
    use_default_device_settings: true
    # if you use_default_device_settings: true, the following settings are ignored.
    # ROS driver will use the default device settings of the camera.
    exposure_auto: true
    exposure_target: 187
    gain_auto: true
    gain_target: 1
    gamma_target: 1.0